processing/src/main/java/io/druid/query/groupby/GroupByQueryQueryToolChest.java

```diff
@@ -171,16 +171,12 @@ public GroupByQueryQueryToolChest(
         // Inject outer query context keys into subquery if they don't already exist in the subquery context.
         // Unlike withOverriddenContext's normal behavior, we want keys present in the subquery to win.
         final Map<String, Object> subqueryContext = Maps.newTreeMap();
-        if (query.getContext() != null) {
-          for (Map.Entry<String, Object> entry : query.getContext().entrySet()) {
-            if (entry.getValue() != null) {
-              subqueryContext.put(entry.getKey(), entry.getValue());
-            }
+        for (Map.Entry<String, Object> entry : query.getContext().entrySet()) {
```

Comment:
The only change here is that an ```if``` clause to check ```query.getContext()``` is null is removed because it cannot be null. I don't want to touch other things because this patch is already big.